src/docfx/lib/json/JsonUtility.cs

```diff
@@ -284,11 +284,6 @@ public static (List<Error>, JToken) RemoveNulls(this JToken root)
 
             RemoveNullsCore(root, null);
 
-            foreach (var (node, name) in nullNodes)
```

Comment:
also remove `nullNodes` traversal in `RemoveNullsCore`

Reply:
sorry, cleaned up more